demo/src/components/McComponent.js

```diff
@@ -134,7 +136,7 @@ render() {
 
 class McOutput extends React.Component {
     render() {
-      const { passage, answer } = this.props;
+      const { passage, answer, attention, question_tokens, passage_tokens } = this.props;
```

Comment:
This is fine, but if I had wrote this from scratch I would have added more hierarchy to `passage` rather than having a separate `passage_tokens`.

I.e.,

```
passage: {
  text: "...",
  tokens: [...]
}

Reply:
Yes, I agree that would be better from the demo's perspective.  I'll have to look at the python code and see if it makes things more complicated there.  I think it does, because of how batching interacts with the demo.  Probably still possible, though.

Reply:
Hmm...  Actually, the `passage` comes from the `requestData`, and the `passage_tokens` comes from the `responseData`.  So I _can't_ actually change this in the python code.  Changing it just in the javascript means packing and then immediately unpacking a javascript object, which seems a little redundant.